configuration/postgres/postgres.go

```diff
@@ -162,42 +163,40 @@ func (p *ConfigurationStore) Get(ctx context.Context, req *configuration.GetRequ
 }
 
 func (p *ConfigurationStore) Subscribe(ctx context.Context, req *configuration.SubscribeRequest, handler configuration.UpdateHandler) (string, error) {
-	var pgNotifyChannels []string
```

Comment:
Removing the list as there could be only one channel passed in metadata in one subscribe call.